app/core/views.py

```diff
@@ -647,3 +652,56 @@ def party_politician_delete_view(request, party_name, politician_id):
         ))
 
     return redirect(reverse('party_dashboard', args=[party_name]))
+
+
+class PoliticianRegistrationView(FormView):
+    model = Politician
+    template_name = 'core/candidates/registration.html'
+    form_class = RegistrationForm
+    success_url = '/registration_send_mail/'
+
+    def get_context_data(self, *args, **kwargs):
+        unique_key = self.kwargs['unique_key']
+        try:
+            user = User.objects.get(registrationkey__unique_key=unique_key)
+            user.is_active = True
+            user.save()
+        except ObjectDoesNotExist:
+            self.template_name = '404.html'
+
+    def form_valid(self, form, *args, **kwargs):
+        unique_key = self.kwargs['unique_key']
+        user = User.objects.get(registrationkey__unique_key=unique_key)
+        politician = Politician.objects.create(
+            first_name=form.data['first_name'],
+            last_name=form.data['last_name'],
+            email=form.data['email'],
+            is_active=False,
+            user_id=user.id
+        )
+
+        if politician is not None:
+            self.send_mail(politician)
+
+        return super(PoliticianRegistrationView, self).form_valid(form)
+
+    def send_mail(self, politician):
+        send_mail(
+            _('Freedomvote account link'),
+            _("""
+            Hello,
+
+            You receive the link for your profile on Freedomvote:
+            {}/politician/{}/edit/profile/
+
+            Sincerely,
+            The Freedomvote Team
+            """).format(settings.BASE_URL, politician.unique_key),
+            'freedomvote@freedomvote.ch',
```

Comment:
This needs to be a setting.. Maybe there is already such a setting, so check before you create a new one.